graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlJcrProperty.java

```diff
@@ -313,6 +313,23 @@ public Long getLongValue() {
         }
     }
 
+    /**
+     * @return The binary size of the JCR node as a Long, null otherwise
+     */
+    @GraphQLField
+    @GraphQLName("size")
+    @GraphQLDescription("The binary size of the JCR node as a Long, null otherwise")
+    public Long getSize() {
```

Comment:
Note quite sure this is the best approach to have this here - but that module is kind of unknown territory for me.
Thoughts @AKarmanov @tdraier